app/models/role.rb

```diff
@@ -26,6 +26,10 @@ def self.add_moderator(person)
     find_or_create_by(person_id: person.id, name: "moderator")
   end
 
+  def self.spotlight?(person)
+    exists?(person_id: person.id, name: "spotlight")
```

Comment:
Okay enough drive by change IMO, you know "always leave the place cleaner  than you found it" ;)